inspirehep/bat/pages/create_literature.py

```diff
@@ -131,24 +131,24 @@ def _thesis_info_population(input_data):
     WebDriverWait(Arsenic(), 10).until(
         EC.visibility_of_element_located((By.ID, 'supervisors-0-name'))
     )
-    Arsenic().find_element_by_id('supervisors-0-name').send_keys(
-        input_data['supervisor']
-    )
-    Arsenic().find_element_by_id('supervisors-0-affiliation').send_keys(
-        input_data['supervisor-affiliation']
+    Select(Arsenic().find_element_by_id('degree_type')).select_by_value(
+        input_data['degree-type']
     )
     Arsenic().find_element_by_id('thesis_date').send_keys(
         input_data['thesis-date']
     )
-    Arsenic().find_element_by_id('defense_date').send_keys(
-        input_data['thesis-defense']
-    )
-    Select(Arsenic().find_element_by_id('degree_type')).select_by_value(
-        input_data['degree-type']
-    )
     Arsenic().find_element_by_id('institution').send_keys(
         input_data['institution']
     )
+    Arsenic().find_element_by_id('supervisors-0-name').send_keys(
+        input_data['supervisor']
+    )
+    Arsenic().find_element_by_id('supervisors-0-affiliation').send_keys(
+        input_data['supervisor-affiliation']
+    )
```

Comment:
So, here you are reordering the selections according to the actual form order. This is not wrong, but is unrelated with what you are doing. When that happens, you should split the related changes in their own commits. For example here you have a commit that reorders things around, and one that deletes `defense_date`.